web-client/src/components/RoleInfo/RoleInfo.tsx

```diff
@@ -0,0 +1,81 @@
+import { AlertTwoTone, HeartTwoTone } from '@ant-design/icons';
+import { Button, Col, Row } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import cav from '../../assets/role_cav.png';
+import pin from '../../assets/role_pin.png';
+import TitleWithAddon from '../TitleWithAddon/TitleWithAddon';
+
+const StyledIntro = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px 50px;
+`;
+
+const Container = styled.div`
+  margin-top: 40px;
+`;
+
+const Box = styled.div`
+  background: #ffffff;
+  box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.15);
+  border-radius: 4px;
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 310px;
+  height: 290px;
+`;
+
+const Pin = styled.img``;
+const Cav = styled.img``;
+
+const LoginIntro: React.FC = (): React.ReactElement => {
+  const { t } = useTranslation();
+  // i18n.changeLanguage('pt-PT');
+  return (
+    <StyledIntro>
+      <TitleWithAddon level={2}>{t('roleinfo.sub_title')}</TitleWithAddon>
+      <Container>
+        <Row>
+          <Col span={12}>
+            <Box>
+              <Pin src={pin} />
+
+              <p>{t('roleinfo.info_pin')}</p>
+
+              <Button
+                type="default"
+                shape="round"
+                icon={<AlertTwoTone twoToneColor="#FF7B02" />}
+              >
+                {t('roleinfo.c2a_pin')}
+              </Button>
+            </Box>
+          </Col>
+          <Col span={12}>
+            <Box>
+              <Cav src={cav} />
+              <p>{t('roleinfo.info_cav')}</p>
+
+              <Button
+                type="default"
+                shape="round"
+                icon={<HeartTwoTone twoToneColor="#C0458A" />}
```

Comment:
I think we should extract these colors into something reusable. Probably theme/colors.js. Maybe AntDesign already has color bindings 🤔 .

Reply:
I don't know if the colors we're using fit in their scheme https://ant.design/docs/spec/colors. If now we can just declare them as constants in colors.js.

Reply:
Agree. Note about this : on the Heart icon, the color is not the color mentioned on the Figma file, because the color overlaying would produce a different result than what is expected to be achieved in the Figma file. But sure we should keep it somewhere where it becomes reusable.